examples/shadowmap/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "luma.gl-examples-shadowmap",
+  "version": "0.0.0",
+  "description": "An es6 shadowmap example for luma.gl.",
+  "license": "MIT",
+  "author": "Ib Green <ib@uber.com>",
```

Comment:
As long as we are keeping separate package.json files, I think you should take the full credit for this one.